tests/test_schema.py

```diff
@@ -2562,3 +2562,43 @@ def resolve(self) -> 'Model':  # noqa
         },
         '$ref': '#/definitions/Model',
     }
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 7), reason='schema generation for generic fields is not available in python < 3.7'
```

Comment:
The test doesn't need to use a generic, so can remove the skipif

Reply:
If there's a good reason to add a test for a generic here too, you can do that as a separate test.

Reply:
No there's no reason for it to be generic, so changed the test to work without them